### PR TITLE
"Variants rework" documentation

### PR DESCRIPTION
**Pull request by bdlukaa**
Documentation for #70

### CODE REVIEW

File: lib/src/attributes/variants/variants.utils.dart [hunk 1]
```diff
@@ -132,4 +130,28 @@ class VariantUtils {
...
+    return Variant<T>(SystemVariants.inactive.value);
+  }
+
+  /// Short Utils: title
+  static Variant<T> title<T extends Attribute>() {
+    return Variant<T>(SystemVariants.title.value);
+  }
+
+  /// Short Utils: paragraph
+  static Variant<T> paragraph<T extends Attribute>() {
```
Comment (leoafarias) [inline lib/src/attributes/variants/variants.utils.dart:150]: Don't think these variants should be  here. Maybe they should get defined within the headless component.

File: lib/src/attributes/variants/variants.utils.dart [hunk 2]
```diff
@@ -132,4 +130,28 @@ class VariantUtils {
...
+    return Variant<T>(SystemVariants.active.value);
+  }
+
+  /// Short Utils: inactive
+  static Variant<T> inactive<T extends Attribute>() {
+    return Variant<T>(SystemVariants.inactive.value);
+  }
+
+  /// Short Utils: title
+  static Variant<T> title<T extends Attribute>() {
```
Comment (leoafarias) [inline lib/src/attributes/variants/variants.utils.dart:145]: Same for this variant. I believe any variant for business logic implementation that is not generic for use in the primitives should not be here

File: lib/src/attributes/variants/variants_short.utils.dart [hunk 3]
```diff
@@ -12,7 +12,16 @@ final landscape = VariantUtils.landscape();
 final dark = VariantUtils.dark();
 final light = VariantUtils.light();
 
+final enabled = not(disabled);
 final disabled = VariantUtils.disabled();
 final focus = VariantUtils.focused();
 final hover = VariantUtils.hover();
 final press = VariantUtils.pressing();
+
+final active = VariantUtils.active();
```
Comment (leoafarias) [inline lib/src/attributes/variants/variants_short.utils.dart:21]: What is an active variant also. Primitives don't have the concept of active, maybe this is part of the headless conversation

**Review by leoafarias — CHANGES_REQUESTED** [review]
I think we should hold a bit on the headless as it's getting it's way into the core of mix.

Let's have a conversation on how to separate that

**Comment by bdlukaa** [comment]
@leoafarias do you think #70 should be reverted, then?

**Comment by leoafarias** [comment]
@bdlukaa will review later and maybe just separate.. give me a couple of days

**Comment by leoafarias** [comment]
@bdlukaa can we revert #70, and keep all the headless specific things separate?

**Pull request closed without merging**